src/fusor/fusor.py

```diff
@@ -562,10 +565,16 @@ def _normalized_gene(
         gene_norm_resp = self.gene_normalizer.normalize(query)
         if gene_norm_resp.match_type:
             gene = gene_norm_resp.gene
-            gene_id = gene_norm_resp.normalized_id
             if use_minimal_gene:
-                return Gene(id=gene_id, label=gene.label), None
-            gene.id = gene_id
+                return MappableConcept(
+                    primaryCoding=Coding(
+                        id=gene.primaryCoding.id,
+                        code=gene.primaryCoding.code,
+                        system=gene.primaryCoding.system,
+                    ),
```

Comment:
This can be simplified
```suggestion
                    primaryCoding=gene.primaryCoding,
```